menas/ui/service/MonitoringService.js

```diff
@@ -352,9 +369,10 @@ var MonitoringService = new function() {
           oRun["warnings"] = [];
 
           MonitoringService.processLatestCheckpoint(oRun);
+          MonitoringService.processCeckpoints(oRun);
           MonitoringService.processRunStatus(oRun);
           MonitoringService.processDirSizes(oRun);
-          MonitoringService.processCheckpoints(oRun);
+          MonitoringService.setRawRecordcount(oRun);
```

Comment:
Atum speaks of `RecordCount` or `recordCount`
```suggestion
          MonitoringService.setRawRecordCount(oRun);
```

Reply:
ok, thnx